runtime/zeitgeist/src/parachain_params.rs

```diff
@@ -48,11 +48,12 @@ parameter_types! {
     // Cumulus and Polkadot
     pub Ancestry: MultiLocation = Parachain(ParachainInfo::parachain_id().into()).into();
     pub const RelayLocation: MultiLocation = MultiLocation::parent();
-    pub const RelayNetwork: NetworkId = NetworkId::Kusama;
-    pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
-    pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+    // Have to change "Any" to "Kusama" for mainnet once we have separate runtimes
+    pub const RelayNetwork: NetworkId = NetworkId::Any;
```

Comment:
I don't get this. Can you explain why?

Reply:
I think this is mistake. I don't remember now why I added this and from where I took inspiration. :sweat_smile: 
I will revert this